Web/Assignment 2/mystyle.css

```diff
@@ -0,0 +1,57 @@
+@CHARSET "ISO-8859-1";
+body
+{
+	
+	background-color: #F0F8FF;
+	
+}
+
+.wrapper
+{
+	
+	
+	background-color: white;
+	width: 800px;
+    height: 700px;
+	margin: 0 auto;
+	padding: 4px 7px;
+}
+.header
+{
+	background-color: #F0F8FF;
+	font-family: serif;
+	font-size: 30px;
+	padding: 5px;
+	border: solid 1px;
+	border-color: #00BFFF;
+	color: #00BFFF;
+
+	
+	
+}
+
+.image
+{    
+	
+	background-image: url("Desert.jpg");	
+	margin-top: 10px;
+	width: 150px;
+	height: 150px;
+	float: left;
+	border: outset;
+	border-color: #ffffff;
+    
+	
+}
+.info
+{
+  
+   float: left;
+   margin-left: 50px;
+   width:400px;
```

Comment:
ACTION: As "float:left" is already mentioned therefore let this div consumes rest of the left space in the left of the "image".